frontend/app/src/components/home/BalanceWithRefresh.svelte

```diff
@@ -70,6 +64,8 @@
                 return t.ethBalance?.toFixed(6) ?? "???";
         }
     }
+
+    refresh();
```

Comment:
We _probably_ use this component in scenarios where the user can select the token they are using. I think that's where we would want the refresh call to react to the change in the value of ledger. I'm a bit worried that this change might break that.